content/en/docs/reference/glossary/workload.md

```diff
@@ -1,28 +1,22 @@
 ---
-title: Workloads
+title: Workload
 id: workloads
 date: 2019-02-13
 full_link: /docs/concepts/workloads/
 short_description: >
-   Workloads are objects you use to manage and run your containers on the cluster.
+   A workload is an application running on Kubernetes.
 
 aka: 
 tags:
 - fundamental
-- core-object
-- workload
 ---
-  Workloads are objects you use to manage and run your containers on the cluster.
+   A workload is an application running on Kubernetes.
 
 <!--more--> 
 
-Kubernetes performs the 
-deployment and updates the workload with the current state of the application.
-Workloads include the DaemonSet, Deployments, Jobs, Pods, ReplicaSet, ReplicationController, and StatefulSet objects.
-
-For example, a workload that has a web element and a database element might run the
-database in one {{< glossary_tooltip term_id="StatefulSet" >}} of
-{{< glossary_tooltip text="pods" term_id="pod" >}} and the webserver via
-a {{< glossary_tooltip term_id="Deployment" >}} that consists of many web app
-{{< glossary_tooltip text="pods" term_id="pod" >}}, all alike.
+Various core objects that represent different types or parts of a workload
+include the DaemonSet, Deployment, Job, ReplicaSet, and StatefulSet objects.
 
+For example, a workload that has a Web server and a database might run the
```

Comment:
nit:
```suggestion
For example, a workload that has a web server and a database might run the
```

Following hard on the heels of Kubernetes resources with initial capitals, readers might wonder what a Web is (maybe something a bit like an Ingress?).

Reply:
ack, agreed.

Reply:
:+1: